dxcint/tests/fixtures/resources/mock_category/mock_1_input.json

```diff
@@ -1,3 +1,3 @@
 {
-  "mock_1.in_1": "Hello World!"
+  "stage-common.in_1": "Hello World!"
```

Comment:
I added locked option to RegisteredTest:74 . Otherwise there is this failure:
FAILED dxcint/tests/test_RegisteredTest.py::test_run_wdl - dxpy.exceptions.DXError: DXWorkflow: the given stage identifier mock_1 could not be found as a stage ID nor as a stage name
Is this the correct behaviour of dxCompiler?

Reply:
yep, just default to `locked`, this is how tests are run now. It should work with `wf_name.input_name`